app/views/claims/verified.html.erb

```diff
@@ -5,9 +5,22 @@
     </h1>
 
     <p class="govuk-body">
-      You can now continue with your claim
+      This is your <%= verified_fields(current_claim) %> from your digital identity:
```

Comment:
```suggestion
      These are your <%= verified_fields(current_claim) %> from your digital identity:
```

Reply:
`These are your full name, address, date of birth, and gender from your digital identity` sounds a bit clumsy to me?

Reply:
Maybe. I think it is the "correct" grammar, but it's not a big deal. Feel free to leave.